test/continuum_opacity.jl

```diff
@@ -1,5 +1,135 @@
 include("opacity_comparison_funcs.jl")
 
+"""
+assert_allclose(actual, reference; [rtol], [atol], [err_msg], [error_location_fmt])
+
+Raises an assertion exception when all entries in `actual` and `reference` are not equal up to the
+desired tolerance. In other words, all entries of `actual` and `reference` must satisfy:
+`abs(actual - reference) <= rtol * abs(reference) + atol`
+
+This is inspired by numpy's assert_allclose function. However, we've changed some default values.
+Additionally, in cases where actual and reference don't have equal values but do have the same 
+shape, the locations of the max errors are also printed.
+
+# Keywords
+- `rtol`: The desired relative tolerance
+- `atol`: The desired absolute tolerance
+- `err_msg`: An optional error message that can be printed when the arrays are not close
+- `error_location_fmt`: An optional keyword used to specify a function that creates a string 
+  describing the location in the arrays where values are unequal, given a `CartesianIndex` object.
+  This should not be specified in cases where actual and reference have different shapes.
+"""
+function assert_allclose(actual, reference; rtol = 1e-7, atol = 0.0, err_msg = nothing,
```

Comment:
Does it make sense to move this into its own file in `/test`?  Seems like it might as well be used throughout the test suite if it's defined.

Reply:
Makes sense to me